{"name": "Search without enter the product name", "status": "broken", "statusDetails": {"message": "TypeError: a bytes-like object is required, not 'method'\n", "trace": "  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\behave\\runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"features\\environment.py\", line 26, in after_step\n    allure.attach(context.driver.get_screenshot_as_png,\n    ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n                  name=\"failed_screenshot\",\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^\n                  attachment_type=AttachmentType.PNG)\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\allure_commons\\_allure.py\", line 200, in __call__\n    plugin_manager.hook.attach_data(body=body, name=name, attachment_type=attachment_type, extension=extension)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\pluggy\\_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\pluggy\\_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    raise exception.with_traceback(exception.__traceback__)\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\allure_behave\\listener.py\", line 171, in attach_data\n    self.logger.attach_data(uuid4(), body, name=name, attachment_type=attachment_type, extension=extension)\n    ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\allure_commons\\reporter.py\", line 165, in attach_data\n    plugin_manager.hook.report_attached_data(body=body, file_name=file_name)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\pluggy\\_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\pluggy\\_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    raise exception.with_traceback(exception.__traceback__)\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\allure_commons\\logger.py\", line 48, in report_attached_data\n    attached_file.write(body)\n    ~~~~~~~~~~~~~~~~~~~^^^^^^\n"}, "steps": [{"name": "Given Navigate to home page", "status": "broken", "statusDetails": {"message": "TypeError: a bytes-like object is required, not 'method'\n", "trace": "  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\behave\\runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"features\\environment.py\", line 26, in after_step\n    allure.attach(context.driver.get_screenshot_as_png,\n    ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n                  name=\"failed_screenshot\",\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^\n                  attachment_type=AttachmentType.PNG)\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\allure_commons\\_allure.py\", line 200, in __call__\n    plugin_manager.hook.attach_data(body=body, name=name, attachment_type=attachment_type, extension=extension)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\pluggy\\_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\pluggy\\_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    raise exception.with_traceback(exception.__traceback__)\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\allure_behave\\listener.py\", line 171, in attach_data\n    self.logger.attach_data(uuid4(), body, name=name, attachment_type=attachment_type, extension=extension)\n    ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\allure_commons\\reporter.py\", line 165, in attach_data\n    plugin_manager.hook.report_attached_data(body=body, file_name=file_name)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\pluggy\\_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\pluggy\\_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    raise exception.with_traceback(exception.__traceback__)\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\DELL LATITUDE 7490\\AppData\\Roaming\\Python\\Python313\\site-packages\\allure_commons\\logger.py\", line 48, in report_attached_data\n    attached_file.write(body)\n    ~~~~~~~~~~~~~~~~~~~^^^^^^\n"}, "attachments": [{"name": "failed_screenshot", "source": "dbc020b2-75c0-4773-8806-c78fb4b46de8-attachment.png", "type": "image/png"}], "start": 1737117503937, "stop": 1737117503944}, {"name": "When Search without entering the product name in search filed", "status": "skipped", "start": 1737117506121, "stop": 1737117506121}, {"name": "And Click on search button", "status": "skipped", "start": 1737117506121, "stop": 1737117506121}, {"name": "Then Proper error message should be displayed", "status": "skipped", "start": 1737117506121, "stop": 1737117506121}], "start": 1737117500967, "stop": 1737117506121, "uuid": "59e29f9f-acee-458d-bc0f-f96d540930c8", "historyId": "0187f819ae4806a81aff61295e88d950", "fullName": "To test the search functionalities: Search without enter the product name", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "To test the search functionalities"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}